ST_grid.c

```diff
@@ -168,10 +169,11 @@ extern SW_SOILWAT SW_Soilwat;
 extern SW_SITE SW_Site;
 extern SW_VEGPROD SW_VegProd;
 extern SW_WEATHER SW_Weather;
+extern pcg32_random_t grid_rng; //this file's unique random number generator
 
 //This is Rgroup structure pointer that will read rgroup disturbance value, will be used in
 // grid disturbance
-extern GroupType     *RGroup [MAX_RGROUPS];
+//extern GroupType     *RGroup [MAX_RGROUPS]; // don't need to extern here, because this is done in ST_globals.h
```

Comment:
This looks like legacy code. If RGroup is defined somewhere else we should remove this.

Reply:
Let's hold off for now, but document this as an issue in the milestone gridded_code